### PR TITLE
feat: add maxHeight to CurrencySearchModal

### PR DESCRIPTION
**Pull request by gbugyis**
Adjusts heights on CurrencySearchModal to cap height at 680px (per design) and prevent lists from becoming unreasonably long. 

Note: the Modal component expects `vh` units, so 680 is converted using the useWindowSize hook.

### CODE REVIEW

File: src/components/SearchModal/CurrencySearchModal.tsx [hunk 1]
```diff
@@ -97,11 +98,17 @@ export default memo(function CurrencySearchModal({
     [setModalView, prevView]
   )
 
+  const { height } = useWindowSize()
   // change min height if not searching
   let minHeight: number | undefined = 80
```
Comment (zzmp) [inline src/components/SearchModal/CurrencySearchModal.tsx:103]: Can these be combined into a single `modalHeight`?

File: src/components/SearchModal/CurrencySearchModal.tsx [hunk 2]
```diff
@@ -97,11 +98,17 @@ export default memo(function CurrencySearchModal({
...
+  let maxHeight: number | undefined = 80
   let content = null
   switch (modalView) {
     case CurrencyModalView.search:
+      if (height) {
+        minHeight = Math.min(Math.round((680 / height) * 100), 80)
```
Comment (zzmp) [inline src/components/SearchModal/CurrencySearchModal.tsx:109]: Add an inline comment that this converts from `px` to `vh`.

**Review by zzmp — CHANGES_REQUESTED** [review]

File: src/components/SearchModal/CurrencySearchModal.tsx [hunk 3]
```diff
@@ -97,11 +98,16 @@ export default memo(function CurrencySearchModal({
     [setModalView, prevView]
   )
 
+  const { height } = useWindowSize()
```
Comment (JFrankfurt) [inline src/components/SearchModal/CurrencySearchModal.tsx:101]: nit: `const { height: windowHeight } = useWindowSize()`

Comment (JFrankfurt) [reply src/components/SearchModal/CurrencySearchModal.tsx:101]: since you're dealing with a few different `height`s here

**Pull request merged**